### PR TITLE
Fix: Change SVG advert icons from Black to White in Dark Mode

### PR DESCRIPTION
**Pull request by ArudoPE**
I propose this as a fix to the Thinkful icons, because right now, they're black in dark mode, making them difficult to see. So this is a quick fix making them white in dark mode.

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behaviour
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
  - `Fix` - bug fixes

Complete the following checkbox ONLY IF it is applicable to your PR. You can complete it later if it is not currently applicable:
-   [x] I have verified all tests and linters pass against my changes, and/or I have included automated tests where applicable

<hr>

**1. Because:**
When you're using Dark Mode on the Odin Project, the Thinkful icons are black, making it difficult to see, right now they look like this in Dark mode:
**Before**
before_theodin_icons

**2. This PR:**
I propose to change the color of the icons from black to white as a quick fix to this problem, and they'll look like this after the fix:
**After**
after_theodin_icons

### CODE REVIEW

File: app/assets/stylesheets/dark-mode.css [hunk 1]
```diff
@@ -135,3 +135,6 @@ progress-circle__inner {
 .progress-circle__content {
   color: var(--dark-bg-accent) !important;
 }
+.advert__icon {
```
Comment (KevinMulhern) [inline app/assets/stylesheets/dark-mode.css:138]: Nit: can we include a new line between this rule and the one above, please?

File: app/assets/stylesheets/dark-mode.css [hunk 2]
```diff
@@ -135,3 +135,6 @@ progress-circle__inner {
...
+    filter: invert(1);
```
Comment (KevinMulhern) [inline app/assets/stylesheets/dark-mode.css:139]: Nit: the indentation looks off on this line, we use two spaces across the app.

Comment (KevinMulhern) [inline app/assets/stylesheets/dark-mode.css:139]: Nit: the indentation looks off on this line, we use two spaces across the app.

**Review by KevinMulhern — CHANGES_REQUESTED** [review]
Hey @ArudoPE, thanks for taking the time to do this. Good solution to the problem 👍 

Just a couple of comments to keep the style of the new rule consistent with the rest of our css rules.

File: app/assets/stylesheets/dark-mode.css (hunk 1 above)
Comment (ArudoPE) [reply app/assets/stylesheets/dark-mode.css:139]: Yes, you're right, and now is fixed 👍🏽 .

File: app/assets/stylesheets/dark-mode.css (hunk 2 above)
Comment (ArudoPE) [reply app/assets/stylesheets/dark-mode.css]: Yes, you're right, and now is fixed 👍🏽 .

**Comment by KevinMulhern** [comment]
Thanks @ArudoPE, looks great 💪 

Before merging, can we [squash this to one commit](https://github.com/TheOdinProject/theodinproject/wiki/Squashing-Commits) please?

**Comment by ArudoPE** [comment]
> Thanks @ArudoPE, looks great muscle
> 
> Before merging, can we [squash this to one commit](https://github.com/TheOdinProject/theodinproject/wiki/Squashing-Commits) please?

Ok, I'll try to 👍.

**Comment by ArudoPE** [comment]
> > Thanks @ArudoPE, looks great muscle
> > Before merging, can we [squash this to one commit](https://github.com/TheOdinProject/theodinproject/wiki/Squashing-Commits) please?
> 
> Ok, I'll try to +1.

I'm deeply sorry about this, I get the idea, squash the last 2 commits into a single one, but sadly I can't get how to do it. I'm closing this PR and making a new one with a single commit.

**Pull request closed without merging**